website/docs/development_guides/dev_hello_world.md

```diff
@@ -144,13 +130,15 @@ metadata:
 Having the object successfully added, you can make a Docker image and push it to your registry:
 
 ```bash
+make generate
 make
 make docker-push
 ```
 
 > **Note:**
 >
 > The default `DOCKER_REGISTRY` is `localhost:5000`, which is preset in `hack/kind-cluster-build.sh`. You can overwrite it to any registry to which you have access permission.
+> To run a local registry with KinD, you could follow the [official doc](https://kind.sigs.k8s.io/docs/user/local-registry/).
```

Comment:
https://github.com/chaos-mesh/chaos-mesh/pull/996
in this pr will run the registry automatically, maybe this modified is not necessary.

Reply:
That would be great. Would fix it later